src/parser/warrior/fury/modules/talents/AngerManagement.js

```diff
@@ -18,29 +22,35 @@ class AngerManagement extends Analyzer {
     spellUsable: SpellUsable,
   };
 
+  cooldownsAffected = [
+    SPELLS.RECKLESSNESS.id,
+  ];
+
   totalRageSpend = 0;
-  wastedReduction = { };
-  effectiveReduction = { };
+  wastedReduction = {};
+  effectiveReduction = {};
 
   constructor(...args) {
     super(...args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.ANGER_MANAGEMENT_TALENT.id);
-    COOLDOWNS_AFFECTED_BY_ANGER_MANAGEMENT.forEach(e => {
+    this.cooldownsAffected.forEach(e => {
       this.wastedReduction[e] = 0;
       this.effectiveReduction[e] = 0;
     });
   }
 
   on_byPlayer_cast(event) {
-    if (!event.classResources ||
-      !event.classResources.filter(e => e.type !== RESOURCE_TYPES.RAGE.id) ||
-      !event.classResources.find(e => e.type === RESOURCE_TYPES.RAGE.id).cost) {
+    if (!event.classResources) {
+      return;
+    }
+    const rage = event.classResources.find(e => e.type === RESOURCE_TYPES.RAGE.id);
+    if (!rage || !rage.cost) {
       return;
     }
 
-    const rageSpend = event.classResources.find(e => e.type === RESOURCE_TYPES.RAGE.id).cost / RAGE_NEEDED_FOR_A_PROC;
+    const rageSpend = rage.cost / 10;
     const reduction = rageSpend / RAGE_NEEDED_FOR_A_PROC * CDR_PER_PROC;
```

Comment:
Does this work only in multiples of `RAGE_NEEDED_FOR_A_PROC`? Meaning that 
- 20 rage = 1 sec
- 30 rage = 1 sec
- 40 rage = 2 sec
Or also in "partial CDR", such as
- 20 rage = 1 sec
- 30 rage = 1,5 sec
- 40 rage = 2 sec
? If it's the first one, you need to round down after dividing:
```javascript
const reduction = Math.floor(rageSpend / RAGE_NEEDED_FOR_A_PROC) * CDR_PER_PROC;
```

Reply:
Works like the second way, in "partial CDR" -> 30 rage = 1,5s